src/components/containers/comment.js

```diff
@@ -39,9 +41,31 @@ class Comment extends Component {
     return true;
   };
 
+  // update comment section
+  editComment = (event) => {
+    const { comment, actions } = this.props;
+    const { replyComment } = this.state;
+    const { id } = comment;
+
+    if (event.which === 13 || event.keyCode === 13) {
```

Comment:
`event.which` and `event.keyCode` are deprecated and might fail to work in different browsers so just find an alternative for this.

I recommend [KeyboardEvent.code](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code)

Reply:
Thanks, I'm looking into it